--- a/vrc-get-gui/build.rs
+++ b/vrc-get-gui/build.rs
@@ -71,25 +71,18 @@
 }
 
 fn get_commit_hash() {
-    use std::process::*;
+    use std::io::prelude::*;
 
-    let Ok(output) = Command::new("git")
-        .arg("rev-parse")
-        .arg("HEAD")
-        .stdin(Stdio::null())
-        .stdout(Stdio::piped())
-        .stderr(Stdio::null())
-        .output()
-    else {
-        return;
-    };
+    let Ok(mut output) = std::fs::File::open("../commit_hash.txt")
+    else {
+        return;
+    };
 
-    let Some(hash_value) = (std::str::from_utf8(&output.stdout).ok())
-        .and_then(|x| x.lines().next())
-        .map(|x| x.trim())
-    else {
-        return;
-    };
+    let mut hash_value = String::new();
+    let Ok(_) = output.read_to_string(&mut hash_value)
+    else {
+        return;
+    };
 
     println!("cargo:rustc-env=COMMIT_HASH={}", hash_value);
 }
